app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -518,6 +519,71 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration(isCompletion = false)
   }
 
+  @Test
+  fun testContinueInteractionAnim_openPrototypeExp_checkContinueButtonAnimatesAfter45Seconds() {
+    launch<ExplorationActivity>(
+      createExplorationActivityIntent(
+        internalProfileId,
+        TEST_TOPIC_ID_0,
+        TEST_STORY_ID_0,
+        TEST_EXPLORATION_ID_2,
+        shouldSavePartialProgress = false
+      )
+    ).use {
+      explorationDataController.startPlayingNewExploration(
+        internalProfileId,
+        TEST_TOPIC_ID_0,
+        TEST_STORY_ID_0,
+        TEST_EXPLORATION_ID_2
+      )
+
+      onView(withId(R.id.animation_continue_button)).check(matches(not(isAnimating())))
+      testCoroutineDispatchers.advanceTimeBy(45000)
```

Comment:
This will cause problems for Espresso tests (it's a *really* long wait). We'll probably need to introduce a Dagger configuration value that we can swap out for tests to make sure they don't need to wait the maximum time. See:

https://github.com/oppia/oppia-android/blob/6b58d6a8f677ca67ad13eb2802b864c4a4d0fc02/domain/src/main/java/org/oppia/android/domain/hintsandsolution/HintsAndSolutionConfigModule.kt#L12

and

https://github.com/oppia/oppia-android/blob/6b58d6a8f677ca67ad13eb2802b864c4a4d0fc02/domain/src/main/java/org/oppia/android/domain/hintsandsolution/HintsAndSolutionConfigFastShowTestModule.kt#L8

Alternatively, we could verify the animation only in a local test (i.e. not one in sharedTest). That's probably fine in this case since it isn't *that* important to make sure these run correctly on Espresso (Robolectric should do an okay job here, and we don't yet have a great testing strategy for animations in general).

Reply:
We aren't using espresso anymore for testing animations. A platform parameter has also been added to disable the animation during espresso tests.